component/web/oauth-common/src/test/java/org/gatein/security/oauth/test/TestOAuthPrincipalProcessor.java

```diff
@@ -105,4 +116,29 @@ public void testLinkedInGenerateGateInUser() {
         assertEquals("lastName", user.getLastName());
         assertEquals("displayName", user.getDisplayName());
     }
+
+    public void testprocessOAuthInteraction() throws IOException, ExecutionException, InterruptedException {
+
```

Comment:
This tests the Scribe Java API, we need to test LinkedinProcessorImpl.java instead